src/humans/player.py

```diff
@@ -41,21 +41,22 @@ def bet(self, amount, index_of_the_hand) -> bool:
 
         return False
 
-    def checkSplitIsPossible(self, index_of_hand_to_split) -> bool:
+    def check_split_is_possible(self, index_of_hand_to_split) -> bool:
         """
         Check if a split is possible according to the money bet on the hand and
         the value of the wallet.
         :return: boolean
            - True : split can be done
            - False: split can't be done
         """
-        if (self.checkDoubleBetIsPossible(index_of_hand_to_split)
+        if (
+                self.check_double_bet_is_possible(index_of_hand_to_split)
                 and self.hands[index_of_hand_to_split].hand.checkSplitIsPossible()):
```

Comment:
Can you put parenthesis on the line under !?

Reply:
yes I can